composer/algorithms/mixup/mixup.py

```diff
@@ -105,18 +95,19 @@ class MixUp(Algorithm):
     Training in this fashion sometimes reduces generalization error.
 
     Args:
-        num_classes (int): the number of classes in the task labels.
         alpha (float, optional): the psuedocount for the Beta distribution used to sample
             mixing parameters. As ``alpha`` grows, the two samples
             in each pair tend to be weighted more equally. As ``alpha``
             approaches 0 from above, the combination approaches only using
             one element of the pair. Default: ``0.2``.
+        interpolate_loss (bool, optional): Interpolates the loss rather than the labels.
+            A useful trick when using a cross entropy loss. Default: ``True``
```

Comment:
Should warn user about this likely yielding incorrect behavior for other losses. Also not sure it should be on by default, but I'll defer to your judgment there.

Reply:
Added a warning in the documentation of `interpolate_loss` and set the default back to `False`. I agree it makes more sense to not turn this on by default.